driver-sync/src/test/functional/com/mongodb/client/ChangeStreamProseTest.java

```diff
@@ -408,7 +410,7 @@ public void test16GetResumeTokenReturnsIdOfPreviousDocument() {
 
     @Test
     public void test19SplitChangeStreamEvents() {
-        assumeTrue(serverVersionAtLeast(7, 0));
+        assumeTrue(getServerVersion().compareTo(new ServerVersion(asList(6, 0, 9))) >= 0);
```

Comment:
Generally we assume latest patch version in these checks.  So it's fine to just check for 6.0.

Confirmed that CI is configured to use 6.0.9 already: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/download-mongodb.sh#L61

Reply:
Thanks, updated